README.md

```diff
@@ -61,14 +61,46 @@ This value can be overriden using `GP2GP_LOGGING_FORMAT` environment variable.
 Alternatively, an external `logback.xml` with much more customizations can be provided using `-Dlogback.configurationFile` JVM parameter.
 
 ## How to run service:
-* Run `./start-local-environment.sh`
 
-If gradle-wrapper.jar doesn't exist run in terminal:
-* If gradle isn't installed `brew install gradle`
-* Update gradle `gradle wrapper`
+### Using the helper script
+
+Run `./start-local-environment.sh`
+
+### Using docker-compose
+
+```
+cd docker/
+docker-compose build
+docker-compose up -d
+```
+
+### Using the GP Connect Public Demonstrator instead of Wiremock
+
+The adaptor uses our own wiremock for the GP Connect API by default. To use the GP Connect public demonstrator instead
+you can use the provided override file for Docker Compose.
+
+```
+cd docker/
+docker-compose build
+docker-compose -f docker-compose.yml -f docker-compose-gpc-pub-demo.yml up -d
```

Comment:
I know I did this originally, but I think we should drop this separate `docker-compose-gpc-pub-demo.yml` file favour of the var.sh approach we took for the proxy story. The latter is much more flexible. Don't need to do it here, I'll create a tech debt story for it.